odl/solvers/advanced/proximal_operators.py

```diff
@@ -568,6 +588,33 @@ def proximal_cconj_l2(space, lam=1, g=None):
 
     Notes
     -----
+    The functional :math:`F` is given by
+
+        .. math::
+
+            F(x) = \\lambda ||x - g||_2
+
+    The convex conjugate :math:`F^*` of :math:`F` is given by
+
+        .. math::
+
+            F^*(y) = \\left\{ \\begin{array}{ll}
+            0 & \\quad  ||y-g||_2 \leq \\lambda, \\\\
```

Comment:
I don't think so... page 10 and 11 [here](http://www.seas.ucla.edu/~vandenbe/236C/lectures/conj.pdf) says that should not be the case. I think it is for `|| . ||_2^2` you have it.